test/test_helper.exs

```diff
@@ -14,7 +14,7 @@ alias Ecto.Integration.TestRepo
 Application.put_env(:ecto, TestRepo,
                     adapter: Mongo.Ecto,
                     url: "ecto://localhost:27017/ecto_test",
-                    pool_size: 1)
+                    size: 1)
```

Comment:
The option is up to each pool adapter. Are the driver's adapters used or are we creating new ones?